configuration/external-storage.md

```diff
@@ -1,99 +1,110 @@
-# External Storage configuration
+# External storage configuration
 You can connect your external hard disk drive (HDD) to any of the USB ports on the Raspberry Pi, and mount the file system to access the data stored on your HDD. 
```

Comment:
Is it worth clarifying that this is e.g. "external hard disk drive or USB flash drive", since I believe from the Raspbian-side of things they work the same?

Reply:
I've added a bit at the start to specify hard disk, SSD and USB flash drive, then replaced HDD with 'storage device' (or 'device' for subsequent uses in the same paragraph to keep verbage down).